Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,23 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose_using_dots
+            last_dot = Time.at(0)
+            while buf = rd.gets
+              log.puts buf
+              # make sure dots printed with interval of at least 30 sec.
+              if Time.now - last_dot > 30
```

Comment:
Does this maybe need to be `(Time.now - last_dot) > 30`? I can never remember the operator precedence.